### PR TITLE
tests: move unit test to a unit test folder

### PR DESCRIPTION
**Pull request by ParthS007**
This refactoring is done to avoid the overriding of env variables in e2e testing.

### CODE REVIEW

**Comment by pamfilos** [comment]
please rebase on top of  `master` branch

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,11 +2,56 @@ name: CI
 
 on:
   push:
-    branches: master
+    branches: [master, refactor]
```
Comment (pamfilos) [inline .github/workflows/ci.yml:5]: remove `refactor`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -26,5 +71,11 @@ jobs:
           pip install --upgrade pip setuptools wheel
           pip install -e .[all]
 
-      - name: Run unit tests
-        run: pytest tests/unit
+      - name: Run pytest
+        run: ./run-tests.sh --check-pytest
```
Comment (pamfilos) [inline .github/workflows/ci.yml:75]: update to run only `unit` tests for now

File: run-tests.sh [hunk 3]
```diff
@@ -23,8 +23,48 @@
...
+
+check_flake8 () {
+    flake8 .
+}
+
+check_manifest () {
+    check-manifest
+}
+
+check_pytest () {
```
Comment (pamfilos) [inline run-tests.sh:48]: add 2 more functions for `check_pytest_unit` and `check_pytest_e2e` so we can run them separately

**Review by pamfilos — CHANGES_REQUESTED** [review]

File: setup.py [hunk 4]
```diff
@@ -23,8 +23,6 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 """CAP client"""
 
-from __future__ import absolute_import, print_function
-
```
Comment (pamfilos) [inline setup.py:27]: remove also `flake8` and `pytest-flake8`, previously introduced in ad1f580

File: run-tests.sh (hunk 3 above)
Comment (ParthS007) [reply run-tests.sh]: added `check_pytest_unit`, what would be the command to run the `check_pytest_e2e`?

**Review by pamfilos — APPROVED** [review]

**Pull request merged**